modules/gcp_bigquery_big_lake_connection/main.tf

```diff
@@ -0,0 +1,13 @@
+# ######################################################################
+# Creates a BigQuery connection to support BigLake table
+# ######################################################################
+
+resource "google_bigquery_connection" "big_lake_connection" {
+  connection_id = var.connection_id
+  description   = var.description
+  friendly_name = var.connection_id
+  location      = var.location
+  provider      = google
+
+  cloud_resource {}
```

Comment:
This is optional and not used, should be deleted unless somehow is needed?

Reply:
There are many types of connections `cloud_resource` must be defined to use the Big Lake table.
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_connection#example-usage---bigquery-connection-cloud-resource

Reply:
<img width="909" alt="image" src="https://user-images.githubusercontent.com/126650298/233605346-d9703f30-5041-4f35-bf67-082cf3d7aa3c.png">